Misc/NEWS.d/3.9.0a1.rst

```diff
@@ -678,7 +678,7 @@ The :keyword:`assert` statement now works properly if the
 
 Removed object cache (``free_list``) for bound method objects.  Temporary
 bound method objects are less used than before thanks to the ``LOAD_METHOD``
-opcode and the ``_PyObject_VectorcallMethod`` C API.
+opcode and the ``PyObject_VectorcallMethod`` C API.
```

Comment:
I don't think that you should modify 3.9.0a1 changelog. I would prefer to only document that functions are renamed in a more recent Changelog (NEWS) entry.

Reply:
Indeed, this was a rebase mistake. Thanks for catching it!